examples/feature_demo/input_fps.py

```diff
@@ -0,0 +1,129 @@
+"""
+First person shooter controls with GLFW GUI
+=========================================
+
+Example showing how to use the Input class to implement FPS controls.
+"""
+
+# sphinx_gallery_pygfx_render = True
+# sphinx_gallery_pygfx_target_name = "disp"
+
+import glfw
+import numpy as np
+import pylinalg as la
+import pygfx as gfx
+from wgpu.gui.glfw import WgpuCanvas, run
+
+
+canvas = WgpuCanvas()
+renderer = gfx.renderers.WgpuRenderer(canvas)
+scene = gfx.Group()
+
+# hide the cursor, lock it to the window and keep it centered
+glfw.set_input_mode(canvas._window, glfw.CURSOR, glfw.CURSOR_DISABLED)
```

Comment:
Nice! Can we also listen to Esc to free the mouse again?

Reply:
Just played with it and I agree :laughing:

Reply:
Haha, how often do you play shooter video games? Usually you alt tab out to get the mouse back. But yes, I can try to implement that.

Reply:
Just tried and alt+tab works! my gaming muscle memory is usually not present when working and I just panic instead :P

Reply:
Added some code to release the mouse with escape (and ignore mouse events while it is released). Clicking on the window captures it again.